front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -46,8 +46,10 @@ const StdcmView = () => {
     useOsrdConfActions() as StdcmConfSliceActions;
 
   const selectedSimulation = simulationsList[selectedSimulationIndex];
-  const isCalculationFailed = isRejected && !isStdcmResultsEmpty;
-  const showResults = !isPending && (showStatusBanner || simulationsList.length > 0);
+  const hasConflicts = (stdcmTrainConflicts?.length ?? 0) > 0;
+  const isCalculationFailed = isRejected;
```

Comment:
This doesn't seem very useful, we could directly use `isRejected` or you can do `isRejected: isCalculationFailed,` l.36 if you prefer